infrastructure/core/template.yaml

```diff
@@ -397,6 +397,38 @@ Resources:
       AliasName: !Sub alias/${AWS::StackName}/${Environment}/csls-kms-key
       TargetKeyId: !Ref CslsKmsKey
 
+  ReplayFromAthenaEcrKmsKey:
+    Type: AWS::KMS::Key
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      EnableKeyRotation: true
+      KeyPolicy:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS: !Sub arn:aws:iam::${AWS::AccountId}:root
+            Action:
+              - kms:*
+            Resource:
+              - '*'
+          - Effect: Allow
+            Principal:
+              Service: ecr.amazonaws.com
+            Action:
+              - kms:Decrypt
+              - kms:Encrypt
+              - kms:GenerateDataKey*
+              - kms:DescribeKey
+            Resource: '*'
+
+  ReplayFromAthenaEcrKmsKeyAlias:
```

Comment:
the key alias and resource names should probably be generic and follow a similar naming convention to the rest of the keys in this file

Reply:
> According to https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3107258369/How+to+deploy+a+container+to+Fargate+with+secure+pipelines#Step-3%3A-Create-a-repository-in-ECR we should have a separate repository per image, so I included the name of the repository/image in the name of the kay - @nacnudus 

ignore this